webpack.config.js

```diff
@@ -0,0 +1,63 @@
+const path = require('path');
+
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+// The path to the ceisum source code
+const cesiumSource = 'node_modules/cesium/Source';
```

Comment:
To make this compile currently, I point this path to my cesium repo `../cesium/Source`, and checkout this branch with the relevant cesium changes: https://github.com/AnalyticalGraphicsInc/cesium/pull/5861